src/liberty/devCommands.ts

```diff
@@ -377,7 +377,8 @@ export async function customDevMode(libProject?: LibertyProject | undefined, par
                 promptString = localize("specify.custom.parms.gradle");
             }
 
-
+            // set focus on the Inputbox
+            await vscode.commands.executeCommand( 'workbench.action.focusNextGroup')
```

Comment:
recommend using `;` at the end of the line to follow the standard across the project